.github/workflows/publish-package.yml

```diff
@@ -1,45 +1,44 @@
-name: Publish package to npm
-
 on:
-  workflow_dispatch:
+  push:
     branches:
       - main
-    inputs:
-      bump:
-        description: 'bump type, major or minor or patch'
-        default: 'patch'
 
 jobs:
-  release:
+  publish:
+    if: github.ref_type == 'branch' && contains(github.ref_name, 'releases/')
```

Comment:
With `on: push` I'm not sure if there's a way to omit merges from PRs other than manually checking `ref_type` and `ref_name`. Ideas welcome.